### DISCUSSION TITLE
New website design

### DISCUSSION THREAD

**Discussion by ocots** [control-toolbox/control-toolbox.github.io#45] [Announcements]
Please, take a look to our new website design: https://control-toolbox.org

@jbcaillau @PierreMartinon @joseph-gergaud @BaptisteCbl

**Comment by jbcaillau** [comment]
awesome @ocots 🫢👍🏽

a few things:
- replace the (times?) font of the title by the standard (mixing the two looks a bit weird)
![image](https://github.com/control-toolbox/control-toolbox.github.io/assets/62183989/aeccae85-e7dc-46dc-87fc-8cc0f7530516)
- not sure it makes sense to show the hierarchy of repos here
![image](https://github.com/control-toolbox/control-toolbox.github.io/assets/62183989/a2740e16-dd28-4795-bb24-ba54671410e4)
(whereas it is informative in the `README` of the main repo)
- @PierreMartinon the certificate of [bocop.org](https://bocop.org) site needs to be updated

**Reply by ocots** [reply]
Done!

**Reply by jbcaillau** [reply]
@ocots @PierreMartinon @joseph-gergaud we need to revise a few things (*cf.* current issues in the `ct` repos), then make an 1.0 release and go to the general registry.

**Comment by jbcaillau** [comment]
@ocots [Documenter 1.0](https://documenter.juliadocs.org/stable) is out: check [this post](https://discourse.julialang.org/t/ann-documenter-1-0/103888)

**Comment by jbcaillau** [comment]
@ocots a few things regarding the documentation:
- Documenter does the job nicely (but rather heavy to run with github CI)
- a bit worried about the possible removal of `$...$` for inline math (whereas these are now standard in github, see [this post](https://github.blog/2022-05-19-math-support-in-markdown))
- more generally the more standard markdown we can use, the better it is (as with [MathJax](https://math.meta.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference))
- not like in Hugo where `\\( ... \\)` must be used for inline math 💩(see [here](https://www.mrnice.dev/posts/how-to-add-math-expressions-to-hugo-blog-the-shortest-guide-possible))
